src/drivers/px4fmu/fmu.cpp

```diff
@@ -838,8 +877,10 @@ PX4FMU::publish_pwm_outputs(uint16_t *values, size_t numvalues)
 void
 PX4FMU::work_start()
 {
+#ifndef PX4FMU_TASK
```

Comment:
I would put the `#ifndef` around the whole function. That way you can't accidentally call it.